tethys_apps/base/app_base.py

```diff
@@ -1085,7 +1085,7 @@ def get_persistent_store_connection(cls, name, as_url=False, as_sessionmaker=Fal
 
         Args:
           name(string): Name of the PersistentStoreConnectionSetting as defined in app.py.
-          as_url(bool): Return SQLAlchemy URL object instead of engine object if True. Defaults to False.
+          as_re_path(bool): Return SQLAlchemy URL object instead of engine object if True. Defaults to False.
```

Comment:
Opps. This change is not necessary I hope. :)

Reply:
Good catch!